packages/shared/src/styles/components/buttons.css

```diff
@@ -10,7 +10,7 @@
   & .icon {
     width: 1em;
     height: 1em;
-    font-size: 1.5rem;
+    font-size: 1.75rem;
```

Comment:
So this change is going to affect other places where this class is used and they will be bigger, right? Maybe we can remove className from the icons above and use on the available sizes (size prop)

Reply:
The icon class is only valid for icons inside buttons.
The other ones it doesn't affect too. But we might want to re-evaluate the button icons globally (see tsahi comment: https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1655216371347519?thread_ts=1655214705.501559&cid=C01L0QXQJNB)